pkg/networkmanager/team.jsx

```diff
@@ -200,7 +200,7 @@
                         <TextInput id={idPrefix + "-ping-interval-input"} className="network-number-field" value={pingInterval} onChange={setPingInterval} />
                     </FormGroup>
                     <FormGroup fieldId={idPrefix + "-ping-target-input"} label={_("Ping target")}>
-                        <TextInput id={idPrefix + "-ping-target-input"} value={pingTarget} onChange={setPingTarget} />
+                        <TextInput id={idPrefix + "-ping-target-input"} value={pingTarget} onChange={(_, val) => setPingTarget(val)} />
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18739-20230503-135116-e27cc6bb-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)